invenio_app_rdm/records_ui/views/__init__.py

```diff
@@ -47,6 +47,19 @@ def create_blueprint(app):
         view_func=record_latest,
     )
 
+    with app.app_context():
```

Comment:
``app.extensions['invenio-rdm-records'].records_service.config`` to avoid pushing an app context